### ISSUE TITLE
[feature] Build a .csv to download and import

### ISSUE THREAD

**Issue by o68x** [tvler/streetpass#13]
I check in mastodon only once in a while, so it would be useful to be able to bulk import the list from a downloaded csv file.

**Comment by tvler** [comment]
I think this is a great idea! leaving this doc here for when I get to it

**Comment by tvler** [comment]
ya like an import/export feature? This could probably be done in the same pass :)

**Comment by girlmaya** [comment]
Yeah. Honestly i'm fine building up the list again instead of just importing, but someone will find almost certainly find it useful

**Comment by jaz-on** [comment]
Hello there, I'm using the StreetPass extension on Firefox for a few weeks.
I just clicked the extension and the counter said I have a solid list of 160+ mastodon accounts were found when I was browsing the web. How cool! I didn't have anything to do! The extension found all these lovely accounts for me! 🙌

Sadly, when I clicked the extension, the interface didn't provide an easy option to open all those profiles, I now have to click all the links one by one, which would be time-consuming with a hundred+ links stored.

A simple option to open all links at once and/or to export a list, so we can import it quickly on Mastodon directly, as @girlmaya suggested.

How realistic is it for this feature to be implemented? :)
In the meantime, is there a simple location where this data is stored that can be found in the Firefox installation files, for example?
